compiler/resolution/implementForallIntents.cpp

```diff
@@ -1374,12 +1378,16 @@ static void propagateExtraLeaderArgs(CallExpr* call, VarSymbol* retSym,
       ArgSymbol* parentOp = eFormal; // the reduceParent arg
       VarSymbol* currOp   = new VarSymbol(astrArg(ix, "reduceCurr"));
       VarSymbol* svar     = new VarSymbol(astrArg(ix, "shadowVar"));
+      VarSymbol* stemp    = newTemp("svTmp");
       redRef1->insertBefore(new DefExpr(currOp));
       redRef1->insertBefore("'move'(%S, clone(%S,%S))", // init
```

Comment:
No, I can't. Things work the same way - i.e. valgrind reports no errors and leaks are the same amount - regardless of whether an autoCopy is present here or not. (I just checked.)